cmd/preview.go

```diff
@@ -105,7 +103,7 @@ func newPreviewCmd() *cobra.Command {
 				return result.FromError(err)
 			}
 
-			m, err := getUpdateMetadata("", root)
+			m, err := getUpdateMetadata(message, root)
```

Comment:
This is a change to align with `up` that's a bit unrelated, but I made as part of doing a pass to align these.  @chrsmith Is this change correct, or is it intentional that we pass an empty message on standalone `pulumi preview` (vs. on the preview associated with a `pulumi up`)?

Notably - we support a `--message` parameter to `pulumi preview` but were previously ignoring it - so I'm nearly certain this was just wrong.

Reply:
Yes, passing the empty string/ignoring it is just plain wrong.

This is the seldom used, but super-useful feature where you can provide a custom message to an update operation. (Since we default to the git commit message, it isn't going to be useful for the local development story if you run `pulumi up` several times from the same commit.) So `pulumi up -m "Add IAM role permissions"` will make matching the actual update correct in the console.

If we can hook that up for both previews and updates, that would be great. (Though in theory, we'd probably want to do it for refreshes and destroys too... Feel free to file an issue and I can make the cleanup.)

Reply:
>  Though in theory, we'd probably want to do it for refreshes and destroys too... 

It appears we already do this correctly for refresh and destroy - it looks like `pulumi preview` was the only one that had this wrong.